iib/workers/tasks/build.py

```diff
@@ -840,6 +901,42 @@ def handle_add_request(
             (prebuild_info['distribution_scope'] in ['dev', 'stage']),
         )
 
+        # Add the max ocp version property
+        # We need to ensure that any bundle which has deprecated/removed API(s) in 1.22/ocp 4.9
+        # will have this property to prevent users from upgrading clusters to 4.9 before upgrading
+        # the operator installed to a version that is compatible with 4.9
+
+        # Get the CSV name and version (not just the bundle path)
+        db_path = temp_dir + "/database/index.db"
+        port, rpc_proc = _serve_index_registry(db_path)
```

Comment:
I'm not sure I follow. My reasoning is that we need the new bundles' `csvName` and `version` in order to add the properties to the index. Instead of untaring the bundle images and parsing the CSV of these bundles I thought I would query the database after they were added so I could get a nice json representation and extract these fields.

Reply:
I see the reason for the second call now. We need to serve the index again, because we want to query for the information of the newly added bundles.